apps/notifications/style.scss

```diff
@@ -6,7 +6,7 @@
 
 #wpnc-panel {
 	position: fixed;
-	top: 47px;
+	top: var( --masterbar-height );
```

Comment:
I wonder if changes to apps would be better as an Issue. Otherwise you're going to get caught up in building and deploying the Notifications app (cc @marekhrabe who knows about this) instead of focusing on shipping Calypso UI code.

Reply:
We will eventually need to touch this file. Not sure why we need to deploy the Notifications app right now? I think it can be deployed when someone needs it updated. As far as calypso is concerned, there is no need for deploy.

Reply:
This is interesting because we might need to alter the behavior of the notifications app in iframe based on the flag of the site it's being embedded it. This might get pretty complex I'm afraid 😬

Reply:
@marekhrabe Thanks for providing PCYsg-elI-p2, now I got it! Eventually, I checked the frontend notifications 
![](https://cln.sh/qhKwnU+)
and they don't depend `#wpnc-panel`. In fact `#wpnc-panel` only exists in calypso context. Notifications in the frontend are working already great and this change will not affect them.